cloud-platform/app-deploy.md

```diff
@@ -0,0 +1,214 @@
+
+Amazon's ECR, within ECS is where all of the application images used by the Cloud Platform are stored.
+
+### Creating or choosing a repository
+
+**NOTE:** *When you are uploading your image to ECR you'll need a credential from the cloud platform team, ask in the `#cloud-platform-users` channel in Slack*
+
+On the ECR start page you will have the option create a new repository, or to search for an existing one.
+
+Create a new repository, and name it something relevant:
+
+![Image](images/create-ecr-repo.png)
+
+After confirmation that the repository was successfully created, ignore the list of commands and click **Done**, at the bottom of the page.
+
+### Authenticating with the repository
+
+Select your newly created repository from the displayed list.
+
+Within the repository, you will see the following section with values unique to you:
+
+![Image](images/repo-values.png)
+
+Click the **View Push Commands** button.
+
+You will then be presented with a list of preconfigured terminal commands for you to run.
+
+Copy the first command. You will need to add `--profile yourAWSProfile` if the AWS account you're pushing to is not the default in your CLI:
+*Note: You can also set the environment variable `AWS_ENVIRONMENT=<yourAWSProfile>`*
+
+`aws ecr get-login --profile mojdsd --no-include-email --region eu-west-1`
+
+Execute the command, then proceed to execute the Docker login command provided in the Terminal.
+
+`Login Succeeded` will confirm you have been authenticated with the repository.
+
+### Pushing the Docker image to the repository
+
+Ensure the Docker image for your application has been built and is stored locally on your machine.
+
+Now we need to tag the image with the tag provided by ECR, so it can be pushed into the correct repository.
+
+View the **Push Commands** again, and modify the fourth command provided, ensuring the first tag is the one currently used on your machine:
+
+`docker tag cloud-platform-demo-app:latest 926803513772.dkr.ecr.us-west-1.amazonaws.com/cloud-platform-demo-app:latest`
+
+Finish by running the fifth command provided, to push the image to your repository.
+
+`docker push 926803513772.dkr.ecr.us-west-1.amazonaws.com/cloud-platform-demo-app:latest`
+
+A quick refresh of ECR, and your image should now be displayed.
+
+## Configuring deployment files
+
+To deploy an application to the Cloud Platform, a number of deployment files must first be configured.
+
+These deployment files will make reference to your application's Docker image and handle configurations, such as ports and host resolving.
+
+This guide will analyse the deployment files for `Cloud-Platform-Demo-App`.
+
+The deployment files can be used as a basic template for your application, and can be found at:
+
+[https://github.com/ministryofjustice/cloud-platform-demo-app](https://github.com/ministryofjustice/cloud-platform-demo-app)
+
+### Deployment
```

Comment:
I think it'd be useful to link to the Kubernetes docs for users when we mention k8s resources - in this case, [`Deployment`](https://kubernetes.io/docs/tasks/run-application/run-stateless-application-deployment/)